cmd/dbmigrate/main.go

```diff
@@ -4,35 +4,44 @@ import (
 	"flag"
 	"fmt"
 	"os"
-	"path/filepath"
 	"runtime"
-	"strconv"
+	"strings"
 	"time"
 
 	"github.com/thrasher-corp/gocryptotrader/common"
 	"github.com/thrasher-corp/gocryptotrader/config"
 	"github.com/thrasher-corp/gocryptotrader/core"
 	"github.com/thrasher-corp/gocryptotrader/database"
-	db "github.com/thrasher-corp/gocryptotrader/database/drivers/postgres"
+	"github.com/thrasher-corp/goose"
+
+	dbPSQL "github.com/thrasher-corp/gocryptotrader/database/drivers/postgres"
 	dbsqlite3 "github.com/thrasher-corp/gocryptotrader/database/drivers/sqlite"
-	mg "github.com/thrasher-corp/gocryptotrader/database/migration"
 )
 
+type argsType []string
+
 var (
-	dbConn          *database.Database
+	dbConn          *database.Db
 	configFile      string
 	defaultDataDir  string
 	createMigration string
 	migrationDir    string
+	command         string
+	args            argsType
 )
 
-var defaultMigration = []byte(`-- up
--- down
-`)
+func (i *argsType) String() string {
+	return "hello"
```

Comment:
👋

Reply:
Just wanted to be friendly and say hello

Also discovered a bug with Goose path imports so hold off on the review for a moment :D